src/components/ui/DataGrid/DataGrid.svelte

```diff
@@ -80,6 +80,23 @@
   let previousSelectedRowId: RowId | null = null;
 
   $: {
+    const seen_set = new Set<number>();
```

Comment:
Do you mind using camelCase for variable names? Otherwise, I think this error is super clear and useful! Do you think it'd also be helpful to print out the offending duplicate values as well?